docs/docs/getting-started/faq.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 3
+description: Frequently asked questions and answers.
+title: FAQ
+---
+
+<!-- if updated, make sure the comment in plugins/rehype/rehype-faq/index.js is considered  -->
+
+# Frequently asked questions and answers {#faq}
+
+Here you can find answers to frequently asked questions and common problems about Mapperly.
+
+## Mapperly does not work when I use source generator X.
+
+Chaining source generators is not supported by Roslyn.
+
+## I updated the Mapperly version, but the generated code still looks the same.
+
+Restart the IDE to make it load the new version of Mapperly. This is a bug of the IDE.
```

Comment:
Is this a general IDE problem or just specific for VisualStudio?

Reply:
As far as we know it can also happen in Rider.